src/CMakeLists.txt

```diff
@@ -817,7 +817,7 @@ endif(APPLE)
 target_link_libraries(darktable lib_darktable)
 
 if (WIN32)
-  set_target_properties(darktable PROPERTIES LINK_FLAGS "-mwindows -Wl,-subsystem,windows")
+  set_target_properties(darktable PROPERTIES LINK_FLAGS "-mconsole -mwindows -Wl,-subsystem,console")
```

Comment:
And this allows printing to the `cmd.exe` console while not showing a console when starting the program?

Reply:
Yes :) You can give a try

Reply:
I just tried on Windows 7. When double clicking darktable.exe in the explorer I get the dreaded terminal window.

Reply:
Indeed. There was no terminal windows when I was launching from command prompt. I have a solution, I'll push it shortly.

Reply:
So I have added the `FreeConsole()` call to the part when we decide to use log to file, and hence no more console is needed. This results that at startup you see a console window flash for a second, then disappears, and darktable's main window will appear, and only that window will be visible.
I have tested the following scenarios:
* Launching w/o arguments from command prompt: works, only dt's main window will be visible
* Launching with `--help` from command prompt: works, you see in the console the usage text
* Launching w/o arguments from explorer by double click: works, only dt's main window will be visible
* I have even tested launching with `--help` from explorer by creating a shortcut and double click on that: that is rather useless, but it displays only a console window, with a usage text, but then it closes, so not too much point IMHO.